assets/src/components/activities/adaptive/components/common/PartComponent.tsx

```diff
@@ -35,7 +35,42 @@ const PartComponent: React.FC<AuthorProps | DeliveryProps> = (props) => {
   const [componentStyle, setComponentStyle] = useState<CSSProperties>(initialStyles);
 
   const [customCssClass, setCustomCssClass] = useState<string>(props.model.customCssClass || '');
+  const handleiFrameStylingChanges = (currentStateSnapshot: Record<string, unknown>) => {
```

Comment:
maybe we can also just call this `handleStylingChanges` later if we want to we can add other keys to check for